src/V2/Endpoint/Account/LuckEndpoint.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace GW2Treasures\GW2Api\V2\Endpoint\Account;
+
+use GW2Treasures\GW2Api\GW2Api;
+use GW2Treasures\GW2Api\V2\Authentication\AuthenticatedEndpoint;
+use GW2Treasures\GW2Api\V2\Authentication\IAuthenticatedEndpoint;
+use GW2Treasures\GW2Api\V2\Endpoint;
+
+class LuckEndpoint extends Endpoint implements IAuthenticatedEndpoint {
+    use AuthenticatedEndpoint;
+
+    public function __construct( GW2Api $api, $apiKey ) {
+        parent::__construct( $api );
+
+        $this->apiKey = $apiKey;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function url() {
+        return 'v2/account/luck';
+    }
+
+    public function get() {
+	$result	= $this->request()->json();
+	return (isset($result[0]) ? $result[0]->value : false);
```

Comment:
You indented these 2 lines with a tab. Please change it to spaces to match the other files.